migration/migrator/data/course_tables.sql

```diff
@@ -778,7 +778,8 @@ CREATE TABLE public.polls (
     status text NOT NULL,
     release_date date NOT NULL,
     image_path text,
-    question_type character varying(35) DEFAULT 'single-response-multiple-correct'::character varying
+    question_type character varying(35) DEFAULT 'single-response-multiple-correct'::character varying,
+    student_histogram_release_setting text NOT NULL
```

Comment:
Use `character varying` if what we're going to be storing is a fairly finite length string.

Reply:
Also I think we can rename this to `release_histogram` to save on characters throughout the codebase. We're not going to have some toggle for releasing histogram to other user groups as far as I'm aware.